client/assets/components/navbar.scss

```diff
@@ -0,0 +1,171 @@
+.top-navbar {
+  height: 70px;
```

Comment:
### Critical

I thought we agreed before adim left to make use of `rem` instead of `px`

Reply:
@williamolojede Created a `px-to-rem` function that makes this easy, you can make reference to that here too.